Add callouts for multi domains support in source maps (#9525)
DataDog/documentation: content/en/real_user_monitoring/error_tracking/_index.md

**Note**: Currently only source maps with the `.min.js` extension will work to correctly unminify stack traces in the Error Tracking UI. Source maps with other extensions (for example, `.mjs`, etc.) while accepted will not unminify stack traces. 

> <div class="alert alert-info">A given JavaScript source file can be served from different subdomains depending on the environment (staging, production, ...). You can upload the related source map once and make it work for multiple subdomains by using the absolute prefix path instead of the full url (i.e: specify <code>/static/js</code> instead of <code>https://hostname.com/static/js</code>).</div>

Reviewer: ```suggestion
<div class="alert alert-info">A given JavaScript source file can be served from different subdomains depending on the environment (for example, staging or production). You can upload the related source map once and make it work for multiple subdomains by using the absolute prefix path instead of the full url (specify <code>/static/js</code> instead of <code>https://hostname.com/static/js</code>).</div>
```
